pkg/runner/flags.go

```diff
@@ -26,7 +26,11 @@ func setupFlags(config v1alpha1.ConfigurationSpec) error {
 	if err := flag.Set("test.fullpath", "false"); err != nil {
 		return err
 	}
-	if err := flag.Set("test.count", "1"); err != nil {
+	repeatCount := "1"
+	if config.RepeatCount != nil {
+		repeatCount = strconv.Itoa(*config.RepeatCount)
+	}
+	if err := flag.Set("test.count", repeatCount); err != nil {
```

Comment:
don't set the flag if repeat count is nil

Reply:
done